nautobot_ssot/integrations/infoblox/urls.py

```diff
@@ -0,0 +1,4 @@
+"""Django urlpatterns declaration for Infoblox integration."""
```

Comment:
If we're not doing any specific links in the UI we should probably remove this file.

Reply:
Removed